labman/db/process.py

```diff
@@ -3492,6 +3492,18 @@ def generate_prep_information(self):
                 result['center_project_name'] = ''
                 result['runid'] = ''
                 result['instrument_model'] = sequencing_run[0]
+                result['orig_name2'] = result['orig_name']
+
+                if result['orig_name2'] is not None and study_id is not None:
+                    # strip the prepended study id from orig_name2, but only
+                    # if this is a 'sample' row, and not a 'control' row.
+                    # (captured here w/orig_name2 and study_id not equaling
```

Comment:
Could you change " 'sample' row " to " 'experimental sample' row "?

Reply:
done!